.travis.yml

```diff
@@ -17,8 +17,8 @@ env:
 #    - env: PLONE_VERSION=4.3.x SOLR_VERSION=5.1.x
 before_install:
   - mkdir -p $HOME/buildout-cache/{eggs,downloads}
-  - virtualenv .
-  - bin/pip install --upgrade pip setuptools zc.buildout
+  - virtualenv -p `which python` .
+  - bin/pip install setuptools==33.1.1 zc.buildout==2.9.5
```

Comment:
@tomgross pinning is a good thing. I prefer to have it in requirements.txt, to make sure it is used everywhere (local dev, travis).